src/viam/sdk/components/movement_sensor/server.cpp

```diff
@@ -19,174 +20,124 @@ MovementSensorServer::MovementSensorServer(std::shared_ptr<ResourceManager> mana
-    *response->mutable_coordinate() = result.coordinate.to_proto();
-    response->set_altitude_m(result.altitude_m);
-
-    return ::grpc::Status();
+    return make_service_helper<MovementSensor>("MovementSensorServer::GetPosition", this, request)(
+        [&](auto& helper, auto& movementsensor) {
+            const MovementSensor::position result = movementsensor->get_position(helper.getExtra());
+            *response->mutable_coordinate() = result.coordinate.to_proto();
+            response->set_altitude_m(result.altitude_m);
+        });
 }
 
 ::grpc::Status MovementSensorServer::GetProperties(::grpc::ServerContext* context,
                                                    const GetPropertiesRequest* request,
                                                    GetPropertiesResponse* response) {
-    const auto status = server_wrapper<MovementSensor>(request);
-    if (!status.ok()) {
-        return status.status;
-    }
-
-    MovementSensor::properties result = status.movementsensor->get_properties(status.extra);
-    response->set_linear_velocity_supported(result.linear_velocity_supported);
-    response->set_angular_velocity_supported(result.angular_velocity_supported);
-    response->set_orientation_supported(result.orientation_supported);
-    response->set_position_supported(result.position_supported);
-    response->set_compass_heading_supported(result.compass_heading_supported);
-    response->set_linear_acceleration_supported(result.linear_acceleration_supported);
-
-    return ::grpc::Status();
+    return make_service_helper<MovementSensor>("MovementSensorServer::GetProperties",
+                                               this,
+                                               request)([&](auto& helper, auto& movementsensor) {
+        const MovementSensor::properties result = movementsensor->get_properties(helper.getExtra());
+        response->set_linear_velocity_supported(result.linear_velocity_supported);
+        response->set_angular_velocity_supported(result.angular_velocity_supported);
+        response->set_orientation_supported(result.orientation_supported);
+        response->set_position_supported(result.position_supported);
+        response->set_compass_heading_supported(result.compass_heading_supported);
+        response->set_linear_acceleration_supported(result.linear_acceleration_supported);
+    });
 }
 
 ::grpc::Status MovementSensorServer::GetAccuracy(::grpc::ServerContext* context,
                                                  const GetAccuracyRequest* request,
                                                  GetAccuracyResponse* response) {
-    const auto status = server_wrapper<MovementSensor>(request);
-    if (!status.ok()) {
-        return status.status;
-    }
-
-    std::unordered_map<std::string, float> result =
-        status.movementsensor->get_accuracy(status.extra);
-    response->mutable_accuracy()->empty();
```

Comment:
I removed this line; I don't think it did anything (my guess is it was an attempt to "empty" the `mutable_accuracy` map, but `empty` just returns a boolean describing if the map _is_ empty that we were ignoring.